lib/cosmos/gui/line_graph/lines.rb

```diff
@@ -209,40 +209,40 @@ def single_line_with_x_states
 
     def add_line(text, y, x = nil, y_labels = nil, x_labels = nil, y_states = nil, x_states = nil, color = 'blue', axis = :LEFT, max_points_plotted = nil)
       # Validate y data
-      raise ArgumentError, "GraphView y data was must given in an array-like class" unless y.respond_to?(:[])
-      raise ArgumentError, "GraphView y data cannot be empty" if y.empty?
-      raise ArgumentError, "GraphView y data must be Numeric" unless y[0].kind_of?(Numeric)
+      raise ArgumentError, "TlmGrapher y data must be given in an array-like class" unless y.respond_to?(:[])
```

Comment:
Shouldn't mention TlmGrapher.  LineGraph can be used in other places too.   How about just delete the word entirely?